src/components/progressCircular/progress-circular.scss

```diff
@@ -74,7 +75,7 @@ md-progress-circular {
   }
 
 
-  .md-spinner-wrapper.md-mode-indeterminate  {
+  .md-mode-indeterminate .md-spinner-wrapper {
```

Comment:
The DOM structure had to change to make this happen.  I added an extra container that could be scaled separate from the outer container.